shared/tracker2/assertion/index.tsx

```diff
@@ -380,12 +380,14 @@ class Assertion extends React.PureComponent<Props, State> {
           </Kb.Text>
           <Kb.ClickableBox onClick={items ? this._toggleMenu : p.onShowProof} style={styles.statusContainer}>
             <Kb.Box2 direction="horizontal" alignItems="center" gap="tiny">
-              <Kb.Icon
-                type={stateToIcon(p.state)}
-                fontSize={20}
-                hoverColor={assertionColorToColor(p.color)}
-                color={p.isSuggestion ? Styles.globalColors.black_20 : assertionColorToColor(p.color)}
-              />
+              <Kb.WithTooltip tooltip={(p.state == 'valid' || p.state == 'revoked') && 'Check proof'}>
```

Comment:
🤔 “Check proof” sounds like a command. Maybe “Proof checked”? Also, let's use `===` instead of `==`. Also, also, should the text be different if it was revoked?

Reply:
Hmm, it's an invitation to click though, it's clickable and you get to the proof. So to me it should say "View proof", or "Go to proof".